src/Input/Mouse.php

```diff
@@ -285,14 +285,16 @@ public function find(string $selectors, int $position = 1): self
         $rightBoundary = \floor($element['right']);
         $bottomBoundary = \floor($element['bottom']);
 
-        $positionX = \mt_rand(\ceil($element['left']), $rightBoundary);
-        $positionY = \mt_rand(\ceil($element['top']), $bottomBoundary);
-
-        $this->scrollToBoundary($rightBoundary, $bottomBoundary)
-            ->move(
-                ($positionX - $this->x),
-                ($positionY - $this->y)
-            );
+        $this->scrollToBoundary($rightBoundary, $bottomBoundary);
+
+        $visibleArea = $this->page->getLayoutMetrics()->getLayoutViewport();
+
+        $offsetX = $visibleArea['pageX'];
+        $offsetY = $visibleArea['pageY'];
+        $positionX = \mt_rand(\ceil($element['left'] - $offsetX), $rightBoundary - $offsetX);
```

Comment:
random_int is preferable to mt_rand.

Reply:
I'll address it in another PR, it's not related to the PR as the functions used did not change.